frontend/src/views/camp/NavigationCamp.vue

```diff
@@ -1,100 +1,216 @@
+              <v-list-item-subtitle>
+                {{ user.profile().firstname + " " + user.profile().surname }}
+              </v-list-item-subtitle>
+            </v-list-item-content>
+            <v-list-item-action>
+              <v-icon>mdi-chevron-right</v-icon>
+            </v-list-item-action>
+          </v-list-item>
+          <SidebarListItem
+            title="Meine Lager"
+            icon="mdi-format-list-bulleted-triangle"
+            :to="{ name: 'camps', query: { isDetail: true } }" />
+        </v-list>
+
+        <v-divider />
+
+        <v-list>
+          <v-list-item two-line>
+            <v-list-item-content>
+              <v-list-item-title>{{ camp().name }}</v-list-item-title>
+              <v-list-item-subtitle>{{ camp().motto }}</v-list-item-subtitle>
+            </v-list-item-content>
+          </v-list-item>
+          <v-divider inset i />
+          <SidebarListItem
+            title="Campeinstellungen"
+            icon="mdi-cogs"
+            :to="campRoute(camp(), 'admin')" />
+          <v-divider inset />
+          <SidebarListItem
+            title="Team"
+            icon="mdi-account-group"
+            :to="campRoute(camp(), 'collaborators')" />
+          <v-divider inset />
+          <SidebarListItem
+            title="PDF / Drucken"
+            icon="mdi-file-outline"
+            :to="campRoute(camp(), 'print')" />
+        </v-list>
+        <div class="mt-auto">
+          <v-btn
+            x-large
+            height="56"
+            text
+            tile
+            block
+            class="ec-close-drawer pb-safe"
+            @click="open = false">
+            Menu schliessen
+            <v-icon right>mdi-close</v-icon>
+          </v-btn>
+        </div>
+      </div>
+    </v-navigation-drawer>
+  </div>
 </template>
 
 <script>
 import { campRoute } from '@/router.js'
 import UserMeta from '@/components/navigation/UserMeta.vue'
 import Logo from '@/components/navigation/Logo.vue'
+import UserAvatar from '@/components/user/UserAvatar.vue'
+import SidebarListItem from '@/components/layout/SidebarListItem.vue'
 
 export default {
   name: 'NavigationCamp',
   components: {
+    SidebarListItem,
+    UserAvatar,
     UserMeta,
     Logo
   },
   props: {
     camp: { type: Function, required: true }
   },
+  data () {
+    return {
+      open: false
+    }
+  },
+  computed: {
+    user () {
+      return this.$auth.user()
+    }
+  },
   methods: {
     campRoute
   }
 }
 </script>
 
 <style lang="scss" scoped>
-  .camp--name::v-deep .v-btn__content {
-    width: 100%;
-  }
+.camp--name::v-deep .v-btn__content {
+  width: 100%;
+}
 
-  .v-bottom-navigation--fixed {
-    height: auto!important;
-    min-height: 56px;
-    padding-bottom: env(safe-area-inset-bottom);
-  }
+.v-bottom-navigation--fixed {
+  height: auto !important;
+  min-height: 56px;
+  padding-bottom: env(safe-area-inset-bottom);
```

Comment:
Again, `env(safe-area-inset-anything)` is just equal to zero on rectangular displays, isn't it?

Reply:
-.- :apple: